File: examples/availability/publish_ipfs/publish_ipfs.php
<?php

require "./vendor/autoload.php";

use Bloock\Bloock;
use Bloock\Client\AvailabilityClient;
use Bloock\Client\RecordClient;
use Bloock\Entity\Availability\IpfsPublisher;

Bloock::$apiKey = getenv("API_KEY");

$recordClient = new RecordClient();
$availabilityClient = new AvailabilityClient();

$record = $recordClient->fromString("Hello world")->build();

$id = $availabilityClient->publish($record, new IpfsPublisher());

print "Record was published successfully\n";

File: examples/availability/publish_ipfs/publish_ipfs.py
import os
import bloock
from bloock.client.record import RecordClient
from bloock.client.availability import AvailabilityClient
from bloock.entity.availability.ipfs_publisher import IpfsPublisher

if __name__ == "__main__":
    bloock.api_key = os.environ["API_KEY"]

    record_client = RecordClient()
    availability_client = AvailabilityClient()

    record = record_client.from_string("Hello world").build()

    id = availability_client.publish(record, IpfsPublisher())

    print("Record was published successfully")

